tests/integration/cases/micronaut-projects_micronaut-test/test.yaml

```diff
@@ -2,25 +2,40 @@
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/.
 
 description: |
-  Test configurations for micronaut-projects/micronaut-test.
+  Test micronaut-projects/micronaut-test. qos-ch/slf4j and ben-manes/caffeine were originally used as manually specified dependencies.
+  We still keep it here even though manual dependencies specification is not available anymore.
```

Comment:
```suggestion
  We still keep it here even though manual dependency specification is not available anymore.
```

Reply:
Fixed in https://github.com/oracle/macaron/pull/844/commits/d00d4827d3f0f998788f91d9eadbe24a488464ca